src/pages/Home/Home.layout.vue

```diff
@@ -77,7 +77,7 @@ import { HomeHeader, HomeList } from './components';
 import { ProviderCard, EventCard } from '../../components';
 import { Event, EventType, Provider } from '../../utils';
 import { handleUserData } from './utils/handleUserData';
-import DIDLogo from '../../assets/logos/didperu.svg';
+import DIDLogo from '../../assets/logos/didperu_dark.svg';
```

Comment:
You use snake_case here but I noticed other files that uses kebab-case, please use only one style

Reply:
Thanks for notice it, I will check